releasenotes/notes/update-bfgs-optimizer-29b4ffa6724fbf38.yaml

```diff
@@ -0,0 +1,7 @@
+upgrade:
+  - |
+    Deleted the deprecated ``factr`` and ``epsilon`` arguments of the :class:`.L_BFGS_B` optimizer.
```

Comment:
Since factr and epsilon are already removed (albeit not quite!) in #8210, where it has upgrade text for these too already, rather have it twice should we just delete this here.

Reply:
Ah, right! I forgot to update the reno after merging #8210 😄 thanks!